packages/design-systems/src/components/composite/Datagrid/SearchFilter/useGraphQLQuery.ts

```diff
@@ -113,6 +113,29 @@ export const useGraphQLQuery = <TData>(props: UseGraphQLQueryProps<TData>) => {
     [valueConverter],
   );
 
+  const convertedSystemFilter = useMemo(() => {
+    const convertedSystemFilter: QueryRow = {};
+    if (systemFilter) {
+      Object.keys(systemFilter).flatMap((key) => {
+        const filterValue = systemFilter[key];
+        const condition = Object.keys(filterValue)[0];
+        const value = Object.values(filterValue)[0] as
```

Comment:
`QueryRowValue` という型定義があるのでそれを使ってください。

```suggestion
        const value = Object.values(filterValue)[0] as QueryRowValue
```